main.tex

```diff
@@ -740,6 +771,16 @@ \subsubsection{Tables}
 the \texttt{SkyCoord} or the \texttt{Time} classes (e.g., converting the
 coordinates to a different frame or accessing the date/time in the desired time scale).
 
+Additionally, there is now interoperability between \texttt{Table}
+objects and \package{pandas} data frames using the \texttt{Table.to_pandas()}
```

Comment:
since you've used `\texttt{Table}`, maybe use `\texttt{DataFrame}`?

Reply:
will do